src/code.h

```diff
@@ -55,6 +55,12 @@ struct CodeState {
 
     // place to store the previous value of STATE(CurrLVars)
     Bag CodeLVars;
+
+    //
+    Obj currFunc;
```

Comment:
This should probably be InitGlobalBad in code.c as well (I'm going on the fact that CodeLVars is, which is mentioned just above)